client/landing/stepper/declarative-flow/flows/site-migration-flow/site-migration-flow.ts

```diff
@@ -58,7 +59,14 @@ const hasSite = ( siteId: number, siteSlug: string ) => {
 
 const siteMigration: FlowV2 = {
 	name: SITE_MIGRATION_FLOW,
-	isSignupFlow: false,
+	get isSignupFlow() {
+		const searchParams = new URLSearchParams( window.location.search );
+		const hasDestinationSite = [
+			searchParams.has( 'siteSlug' ),
+			searchParams.has( 'siteId' ),
+		].some( Boolean );
+		return ! hasDestinationSite;
+	},
```

Comment:
when will this function be invoked?

Reply:
Here https://github.com/Automattic/wp-calypso/blob/0ca19e88bce23a93ff82f7d5a5e32bbb7e13f728/client/landing/stepper/declarative-flow/internals/hooks/use-sign-up-start-tracking/index.tsx#L20


It is used to check if the `calypso_signup_start` should be triggered.

Reply:
nvm, looks like it will be used here https://github.com/Automattic/wp-calypso/blob/abaa74ba38cfd9623b2699bdaa658176e3ac6d8c/client/landing/stepper/declarative-flow/internals/index.tsx#L212